src/main/java/core/basesyntax/model/lib/Injector.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.model.lib;
+
+import core.basesyntax.model.dao.BetDao;
+import core.basesyntax.model.dao.UserDao;
+import core.basesyntax.model.factory.Factory;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+
+public class Injector {
+
+    public static Object getInstance(Class clazz) throws NoSuchMethodException,
+            IllegalAccessException, InvocationTargetException,
+            InstantiationException, NoDaoAnnotationException {
```

Comment:
should we indicate that method throws RuntimeException?

Reply:
done